source/routes/create/daily.js

```diff
@@ -0,0 +1,49 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
+
+/*
+Send in json form of the bullet and it gets sent to the database
+
+The daily entry db will be of form:
+{
+    "user": "dave",
+    "date": "2021-05-09",
+    "docType": "dailyJournal",
+    "monthKey": "May",
+    "bullets": ["bullet-id1", "bullet-id2", ...]
+}
+*/
+module.exports = {
+  '/create/daily': {
+    methods: ['post'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7433333)
```suggestion
        methods: ["post"],
```